x-pack/plugins/ml/public/application/management/jobs_list/components/jobs_list_page/jobs_list_page.tsx

```diff
@@ -141,7 +142,11 @@ export const JobsListPage: FC<Props> = ({
             }}
           >
             <ContextWrapper feature={PLUGIN_ID}>
-              <EnabledFeaturesContextProvider isServerless={isServerless} mlFeatures={mlFeatures}>
+              <EnabledFeaturesContextProvider
+                isServerless={isServerless}
+                mlFeatures={mlFeatures}
+                showMLNavMenu={chromeStyle === 'classic'}
```

Comment:
For this management page, `showMLNavMenu` will not be used, so this could be given a hardcoded value, rather than going to the effort of loading the chrome style.

Reply:
Not sure I understood Can you clarify what hardcoded value I need to insert? cheers

Reply:
This page is used in stack management, it is not part of the main ML app and so doesn't need to know about the value of `showMLNavMenu`.
We use the same context for enabled features as we reuse components which rely on it, but not all of the values in the context are read,  `showMLNavMenu` will not be read and so it doesn't matter what its value is.
You could possibly make this optional in the Props or just hard code `true` or `false` on this line.

Reply:
I see, I'll make it optional then 👍

Reply:
Done in https://github.com/elastic/kibana/pull/178861/commits/65d1559ba2cd867bf57f6350e7416e1428aefce1